modules/custom/az_dam/config/quickstart/field.storage.media.field_media_ib_dam_embed.yml

```diff
@@ -0,0 +1,21 @@
+uuid: 7c6526e4-ef8d-4040-adb3-967c1919ce9a
+langcode: en
+status: true
+dependencies:
+  module:
+    - link
+    - media
+_core:
+  default_config_hash: v0xW1nC9g8gauXZZwTrKrhmhNO4bzuwD5beQ-1udJJ4
```

Comment:
```suggestion
```

Reply:
Resolved locally. Will commit for review when all changes have been made.